### PR TITLE
Design and CSS tweaks

### PR DESCRIPTION
**Pull request by mdo**
Few small changes to button things up while I was writing the v4.3.0 post.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
       <div class="container">
         <a class="bs-docs-booticon" href="/" aria-label="Bootstrap">
         {% include bootstrap-outline.svg %}</a>
-        <h1>The Bootstrap Blog</h1>
+        <h1><a href="/">The Bootstrap Blog</a></h1>
```
Comment (XhmikosR) [inline _includes/header.html]: Is this really needed? We wrap the whole thing already in an `a` tag.

Comment (mdo) [reply _includes/header.html]: Only the `<svg>` is linked currently.

Comment (mdo) [reply _includes/header.html]: Consolidated the anchors so it's one linke.

Comment (XhmikosR) [reply _includes/header.html]: Ah yeah you are right. Your call how you want the hover effect to be then, either way is fine.

**Pull request merged**